### ISSUE TITLE
Structured Data for SEO

### ISSUE THREAD

**Issue by gusnaughton** [AlaskaAirlines/auro-flight#5]
## Is your feature request related to a problem? Please describe.
One of the issues we currently face in our current solution is SEO for our individual experiences. We rely primarily on content pages to handle directing traffic in a method that looks like this:

```
[search aggregator] -----> [content page] ------> [planbook with search params prefilled] --------> [flight results page]
```

In the Everymundo case, we rely on client-side scraping of a page to send live feeds of data to the vendor to display low fares on various pages and optimize, again, for SEO.

Ideally, as an engineer, no additional work would be required to optimize SEO on unauthenticated pages across our digital platform in the instance of a flight result.

## Describe the solution you'd like
Integration of [structured data](https://schema.org/Flight) will allow crawlers to detect instances of a Flight and its state (fares, gate assignments, duration, etc). They will [work this information into their search experiences](https://developers.google.com/search/docs/guides/intro-structured-data) in contextually helpful ways.

## Describe alternatives you've considered
Current implementation is client-side browser scraping, which I do not believe is ideal or sharable.

## Additional context
[Everymundo's guide to structured data](https://www.everymundo.com/seo-for-airlines/structured-data-implementation/)

**Comment by blackfalcon** [comment]
I feel that this is a no-brainer. Is there any additional data we need to capture other than what is already in the API? 
https://github.com/AlaskaAirlines/auro-flight/blob/main/src/auro-flight.js#L47-L53

Other than that, this is simply another template in the element.

**Comment by blackfalcon** [comment]
The work done in the [POC](https://github.com/AlaskaAirlines/auro-flight/pull/23/files#diff-3a328bf6f623de6341227474f92cdd63b0acd61f095d7255ab9cd61417ba531dR99-R111) was very enlightening. The outcome of the POC gives strong encouragement that we should be doing this work. 

Throughout the execution of this feature, I will also be working with the flight search team to see how this update will interact with the flight experience and other data they present in the search results.

**Comment by blackfalcon** [comment]
This issue was addressed with the v2.0 release.
